clients/web/src/views/App.js

```diff
@@ -39,6 +39,7 @@ var App = View.extend({
     initialize: function (settings) {
         this._started = false;
         settings = settings || {};
+        this.contactEmail = settings.contactEmail || '';
```

Comment:
Let's use `null` for the value when no setting is provided, since that will never be confused with a case where a server sends a setting value that's purposely set to empty string.